submissions/amooChristianah/javascript/productlist.js

```diff
@@ -19,4 +19,27 @@ const productList = [
 
 productList.forEach((product) => {
     console.log(product.productName + ' with '+ product.imageUrl + ' is sold for ' + product.productPrice)
-  });
\ No newline at end of file
+  });
+
+
+
+
+/* TASK 2 */
+
+  var x = 123 - 7;
+
+
+for(var i = 1; i <= x; i++){
+	if(i % 3 == 0 && i % 5 == 0){
```

Comment:
I think you should use strict `===` in all cases @pelumi454

Reply:
Please review my PR now